### ISSUE TITLE
Bug third and fourth player trying to play crashes everything and freezes the third and fourth

### ISSUE THREAD

**Issue by jeffabner** [EmulatorJS/EmulatorJS-netplay-old#45]
good night friends, I spent all day obfuscating the codes, hours without result and continuing to test until things work, I am managing to play with the server hosting on my machine, I had problems connecting a third player and a fourth, because it crashed, it was slow, then reading other errors i had the idea of restart the game, and that solves the problem, i will be testing playing 5 players to play bombeman and other games

restart 
![image](https://github.com/EmulatorJS/EmulatorJS-netplay-old/assets/33882384/4c31d8d7-fdf9-4853-9182-63b2e68dfc53)

the bug is this
![image](https://github.com/EmulatorJS/EmulatorJS-netplay-old/assets/33882384/497c410f-fa49-4e1c-afec-4a43094eec59)
![image](https://github.com/EmulatorJS/EmulatorJS-netplay-old/assets/33882384/33e14416-b137-4404-a720-d5502d0b662b)

**Comment by ethanaobrien** [comment]
I'd recommend waiting for the next version of netplay

**Comment by jeffabner** [comment]
Good afternoon, I admire your dedication here in Brazil, I went to check the js code, impossible to see, I used a website to format it and it was visible to obfuscate, debugging and I ended up finding the netplay error and I was able to play, and debugging I managed to circumvent the limit of 2 players per room

https://beautifier.io/ 
![image](https://github.com/EmulatorJS/EmulatorJS-netplay-old/assets/33882384/81ef512f-8ab6-45af-ab91-56cf1e3a4ee2)
![image](https://github.com/EmulatorJS/EmulatorJS-netplay-old/assets/33882384/89f71b1d-8381-4912-b4ba-a9436e9b45f8)
![image](https://github.com/EmulatorJS/EmulatorJS-netplay-old/assets/33882384/d54b49b5-0a16-44db-b07d-0adb70f94266)

**Comment by ethanaobrien** [comment]
Why are you editing the minified files

**Comment by jeffabner** [comment]
I'm using version 31.6 that takes netplay, I tried to make it work without the reduced and it gave an error on the console and it didn't work, if we happen to talk on discord or in the group, I understand javascript and other languages and it could help

**Comment by ethanaobrien** [comment]
Yeah,  just ping me in the emulatorjs discord. The invite link is in the emulatorjs readme

**Comment by jeffabner** [comment]
in fact, the emulator is directing to this reduced one, I tried to rename it to the complete one and it gave the error, I will start to understand the codes more and see the difference of each one

**Comment by ethanaobrien** [comment]
>in fact, the emulator is directing to this reduced one, I tried to rename it to the complete one and it gave the error, I will start to understand the codes more and see the difference of each one

Enable `EJS_DEBUG_XX`
